stored_queries/ncbi_tax/ncbi_taxon_get_siblings.yaml

```diff
@@ -0,0 +1,26 @@
+# Get the array of siblings for a taxon
+# Results are limited to 10k
+name: ncbi_taxon_get_siblings
+params:
+  type: object
+  required: [key]
+  properties:
+    key:
+      type: string
```

Comment:
Test for bad params?

Reply:
Added a test for a nonexistent key
An incorrect type on this field will be caught by the api itself

Reply:
Nonexistent key should probably throw a 404 or some other error. I'm not sure how to do that if it's only AQL. I think this is good enough for now

Reply:
> An incorrect type on this field will be caught by the api itself
A test here would make sure the type is specified correctly though, so if someone accidentally changes the type def a test would fail